frontend/src/common/queries/cellGuide.ts

```diff
@@ -395,6 +395,52 @@ export const fetchTissueMetadata =
     return await response.json();
   };
 
+/* ========== Lookup tables for organs ========== */
+export function useAllOrgansLookupTables(): {
+  organLabelToIdMap: Map<string, string>;
+} {
+  const { data: allOrgansData } = useTissueMetadata();
+  return useMemo(() => {
+    if (!allOrgansData) {
+      return {
+        organLabelToIdMap: new Map<string, string>(),
+      };
+    }
+
+    const allOrgansLabelToIdMap = new Map<string, string>();
+    for (const organId in allOrgansData) {
+      const organData = allOrgansData[organId];
+      allOrgansLabelToIdMap.set(organData.name, organData.id);
+    }
+    return {
+      organLabelToIdMap: allOrgansLabelToIdMap,
+    };
+  }, [allOrgansData]);
+}
+
+/* ========== Lookup tables for tissues ========== */
+export function useAllTissuesLookupTables(cellTypeId: string): {
```

Comment:
Same for this function - just return the thing directly (allTissuesLabelToIdMap)

Reply:
will fix